### PR TITLE
Make Nui tests work. Framereader moved to tools/clib

### PR DESCRIPTION
**Pull request by littlemountainman**
* Adjusted .pro script to find the .hpp file.

* Changed path in TestFramereader to find Framereader in tools/clib

### CODE REVIEW

File: tools/ubuntu_setup.sh [hunk 1]
```diff
@@ -42,8 +42,7 @@ sudo apt-get update && sudo apt-get install -y \
     sudo \
     vim \
     wget \
-    gcc-arm-none-eabi
-
+    gcc-arm-none-eabi
```
Comment (geohot) [inline tools/ubuntu_setup.sh]: Null change, please remove

File: tools/sim/start_carla.sh [hunk 2]
```diff
@@ -15,5 +15,4 @@ if [ ! -d carla ]; then
 fi
 
 cd carla
-./CarlaUE4.sh Town04
-
+./CarlaUE4.sh
```
Comment (geohot) [inline tools/sim/start_carla.sh]: Why change this?

**Comment by geohot** [comment]
Other then those 2 comments, LGTM

File: tools/sim/start_carla.sh (hunk 2 above)
Comment (littlemountainman) [reply tools/sim/start_carla.sh]: It's set via the client, Town04 changes nothing, town gets loaded in bridge.py in line 89.

**Comment by littlemountainman** [comment]
fixed first comment, explained second why Town04 was removed.

**Comment by littlemountainman** [comment]
Can be merged now. imo

**Pull request merged**